Blockzilla/UserDefault.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
```

Comment:
Missing comment 
/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */

Reply:
Updated